apps/api/src/server/app/case-officer/case-officer.controller.js

```diff
@@ -25,7 +27,10 @@ const getAppealDetails = async function (request, response) {
 	return response.send(formattedAppeal);
 };
 
-const confirmingLPAQuestionnaire = function (request, response) {
+const confirmingLPAQuestionnaire = async function (request, response) {
```

Comment:
Could we please add documentation for this endpoint to swagger in the routes file?